### PR TITLE
feat(sdk): add wasmedge runtime

### PR DESCRIPTION
**Pull request by afmika**

### CODE REVIEW

File: libs/common/src/archive.rs [hunk 1]
```diff
@@ -104,3 +104,8 @@ pub fn flat_list_dir<P: AsRef<Path>>(dir: P) -> Result<Vec<String>> {
     }
     Ok(ret)
 }
+
+pub fn encode_to_base_64(file: &Path) -> Result<String> {
+    let bytes = fs::read(file).context(format!("reading file {:?}", file.display()))?;
```
Comment (zifeo) [inline libs/common/src/archive.rs:109]: with_context to lazily build the message?

**Review by zifeo — CHANGES_REQUESTED** [review]
Sweet!

**Comment by zifeo** [comment]
lgtm, I let @Natoandro approve it

**Review by Natoandro — APPROVED** [review]
LGTM

**Pull request merged**